Source/Tools/FEXRootFSFetcher/Main.cpp

```diff
@@ -85,6 +95,15 @@ void ParseArguments(int argc, char** argv) {
     DistroVersion = Options["distro_version"];
   }
 
+  if (Options.is_set_by_user("force_ui")) {
+    auto Option = Options["force_ui"];
+    if (Option == "tty") {
+      UIOption = UIOverrideOption::TTY;
+    } else if (Option == "zenity") {
+      UIOption = UIOverrideOption::Zenity;
```

Comment:
How does zenity decide whether or not to spawn a graphical dialog?

FexRootFSFetcher always prompts via CLI for me, but when invoking `zenity` on its own I'm getting a graphical dialog.

Reply:
Ah right, makes sense. Thanks!